model/yeast-GEM.txt

```diff
@@ -943,13 +943,13 @@ r_1208	L-leucine[v] <=> L-leucine[c] 	YCL038C	-1000.00	1000.00	  0.00
 r_1209	H+[v] + L-leucine[c] => H+[c] + L-leucine[v] 	YJR001W	  0.00	1000.00	  0.00
 r_1210	H+[v] + L-leucine[v] => H+[c] + L-leucine[c] 	YKL146W or YNL101W	  0.00	1000.00	  0.00
 r_1211	H+[e] + L-leucine[e] <=> H+[c] + L-leucine[c] 	YBR068C or YBR069C or YCL025C or YDR046C or YDR508C or YKR039W	-1000.00	1000.00	  0.00
-r_1212	H+[v] + L-lysine[c] => H+[c] + L-lysine[v] 	YOL092W or YBR293W or YCL069W or YMR088C	  0.00	1000.00	  0.00
```

Comment:
Double check this. I cannot see why YMR088C is not a lysine transporter from the annotation link you supplied. But according to the annotation from biocyc [YMR088C](https://biocyc.org/gene?orgid=YEAST&id=G3O-32788), this protein can transport lysine.